ext/Sys-Hostname/t/Hostname.t

```diff
@@ -1,7 +1,7 @@
 #!./perl
 
 BEGIN {
-    require Config; Config->import;
+    use Config;
```

Comment:
here `use Clonfig` is correct, but either we want to move it before the BEGIN either we want to setup `@INC` and preserve the `require + import`